enterprise/coderd/workspaceproxy.go

```diff
@@ -965,12 +965,8 @@ func convertRegion(proxy database.WorkspaceProxy, status proxyhealth.ProxyStatus
 func convertProxy(p database.WorkspaceProxy, status proxyhealth.ProxyStatus) codersdk.WorkspaceProxy {
 	now := dbtime.Now()
 	if p.IsPrimary() {
-		// Primary is always healthy since the primary serves the api that this
-		// is returned from.
-		u, _ := url.Parse(p.Url)
 		status = proxyhealth.ProxyStatus{
 			Proxy:     p,
-			ProxyHost: u.Host,
```

Comment:
Rather than convert `ProxyHost` to the new format, I deleted it as it is not used in this function or returned in any way.  Also any non-primary proxy `status` we pass is already missing `ProxyHost`.  We should probably use a narrower type or something, idk.